src/components/SortAndGenreControl/SortControl.jsx

```diff
@@ -10,14 +11,17 @@ class SortControl extends React.Component {
     const { currentSelection } = this.props;
 
     return (
-      <div>
-        <label htmlFor="sortSelect">Sort by : </label>
+      <div className="sort-container">
+        <label className="label" htmlFor="sortSelect">
+          Sort by :{" "}
```

Comment:
Why do you use here `{" "}`? Could be replaced with the constant, like
`const blankString = " "`

Reply:
Removed `{" "}` and used BlankString constant.